Lesson15/Akan/script.js

```diff
@@ -0,0 +1,407 @@
+     includes `word` (use the .includes() method).
+   - If true, log: "<word> found in sentence."
+   - Else, log: "<word> not found in sentence."
+*/
+
+function containsWorld(sentence, world) {
+  if (sentence.includes(world)) {
+    console.log(world + " found in sentence.");
+  }
+  else {
+    console.log(world + " not found in sentence.");
+  }
+}
+
+/*
+6. File Extension Check
+   - Define a function `checkFileExtension(filename)` that checks if the filename
+     ends with ".pdf" using .endsWith().
+   - If it does, log: "This is a PDF file."
+   - Otherwise, log: "Not a PDF file."
+*/
+
+function checkFileExtension(filename) {
+  if (filename.endsWith('.pdf'))
+    console.log('This is a PDF file.');
+  else
+    console.log('Not a PDF file.');
+}
+
+/*
+8. Palindrome Check
+   - Define a function `isPalindrome(str)` that checks if `str` is the same
+     forwards and backwards (normalize the string and compare it to reversed version of itself).
+   - If it is, log: "<str> is a palindrome"
+   - Otherwise, log: "<str> is not a palindrome"
+*/
+
+function isPalindrome(str) {
+  let normalized = str.toLowerCase().replace(/[^a-z0-9]/g, '');
+  let reversed = normalized.split('').reverse().join('');
+  if (normalized == reversed) {
+    console.log(str + " is a palindrome");
+  }
+  else {
+    console.log(str + " is not a palindrome");
+  }
+}
+
+/*
+10. Check Even or Odd (if-else)
+   - Define a function `evenOrOdd(number)` that:
+     - Logs "Even" if the number is even
+     - Logs "Odd" if the number is odd
+*/
+
+function evenOrOdd(number) {
+  if (number % 2 == 0) {
+    console.log("Even");
+  } else {
+    console.log("Odd");
+  }
+}
+
+/*
+11. URL Protocol Checker
+   - Define a function `checkProtocol(url)` that converts the URL to lowercase
+     and checks if it starts with "https" using .startsWith().
+   - Log "Secure connection" if true, otherwise "Unsecure connection".
+*/
+
+function checkProtocol(url) {
+  if (url.toLowerCase().startsWith("https"))
+    console.log("Secure connection");
+  else
+    console.log("Unsecure connection");
+}
+/*
+13. Repeat a String
+   - Define a function `repeatWord(word, times)` that uses the .repeat() method
+     to repeat `word` `times` times.
+   - Log the repeated result.
+*/
+
+function repeatWorld(world.times) {
+  console.log(repeatWorld(world, times));
+}
+
+
+/*
+14. Replace Substring
+   - Define a function `censorWord(sentence, target)` that replaces `target`
+     with "****" (use .replaceAll() or multiple .replace()).
+   - Log the censored sentence.
+*/
+
+function censorWord(sentence, target) {
+  console.log(sentence.replaceAll(target, '****'));
+}
+
+/*
+15. Check First Character (if-else)
+   - Define a function `startsWithA(str)` that checks if the string starts with 'A'
+     (use .charAt(0) or [0]).
+   - Log "Starts with A" or "Does not start with A".
+*/
+
+function startsWithA(str) {
+  if (str.chartAt(0 == 'A')) {
+    console.log("Starts with A");
+  }
```

Comment:
We also need an else case to print the other message
```suggestion
  } else {
    console.log("Does not start with A");
  }
```